rails_programming/forms_and_authentication/form_basics.md

```diff
@@ -249,6 +241,26 @@ Once the form gets submitted, the `#create` action will build another new User o
 
 If the `@user` cannot be saved, like because the `first_name` contains numbers, we will jump straight back to rendering the `new.html.erb` view, this time using the `@user` instance that will still have errors attached to it.  Our form should gracefully handle those errors by telling the user where they screwed up.
 
+Notice how in our else clause we specify a status
+
+~~~ruby
+else
+  render :new, status: :unprocessable_entity
+~~~
+
+Remember our Turbo Drive lesson? Which status codes does Turbo handle? Either a redirect, a 4XX status or a 5XX status. On a successful save of our created object you can see we do a redirect
+
+~~~ruby
+if @user.save
+  redirect_to @user
+~~~
+
+This is the 303 redirect status we mentioned in the earlier Turbo Drive lesson. If we don't redirect, and we don't want a 5XX error as that means our application has hit an unexpected issue, that only leaves us with a 4XX status code. If we didn't include that `status: :unprocessable_entity` what would be returned? It would actually be 200 OK status because that is the expected response to a failed form submission. The request has completed in the way we expected. Therefore when dealing with a form validation on the server that fails, you need to specify that return status. Doing so will change a `Completed 200 OK in 206ms (Views: 125.5ms | ActiveRecord: 8.1ms | Allocations: 18793)` return status to `Completed 422 Unprocessable Entity in 52ms (Views: 44.6ms | ActiveRecord: 0.0ms | Allocations: 6391)` and as we know, Turbo will handle that and update the HTML with the returned response.
```

Comment:
There's a lot to unpack here, it could do with being broken up a little imo.

```suggestion
This is the 303 redirect status we mentioned in the earlier Turbo Drive lesson. If we don't redirect, and we don't want a 5XX error as that means our application has hit an unexpected issue, that only leaves us with a 4XX status code.

What would be returned if we didn't include `status: :unprocessable_entity`? 

It would be 200 OK status because that is the expected response to a failed form submission. The request has completed in the way we expected. Therefore when dealing with a form validation on the server that fails, you need to specify the 422 HTTP status. Doing so will change a `Completed 200 OK in 206ms (Views: 125.5ms | ActiveRecord: 8.1ms | Allocations: 18793)` return status to `Completed 422 Unprocessable Entity in 52ms (Views: 44.6ms | ActiveRecord: 0.0ms | Allocations: 6391)`, and as we know, Turbo will handle this and update the HTML with the returned response.
```